docs/using.rst

```diff
@@ -67,7 +67,7 @@ contain useful documentation and a quickstart guide.
   * `Vagrantfile <../components/centos/centos-docker-base-setup/Vagrantfile>`_
   * `README <../components/centos/centos-docker-base-setup/README.rst>`_
 
-* Docker and Kubernetes for use with host-based tools or via ``vagrant ssh``
+* Kubernetes for use with host-based tools or via ``vagrant ssh``
```

Comment:
See, https://github.com/projectatomic/adb-atomic-developer-bundle and https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst this was what was decided on after discussion, Usage doc needs to be consistent with Readme and Installation document.